### DISCUSSION TITLE
Appimage with HEIC support (and other delegates)

### DISCUSSION THREAD

**Discussion by richard-llmnn** [ImageMagick/ImageMagick#6692] [Contribute]
### Is your feature request related to a problem? Please describe.

On the [official download page](https://imagemagick.org/script/download.php) of ImageMagick there is only one  appimage version available. The appimage supports the following delegates: `bzlib djvu fontconfig freetype jbig jng jpeg lcms lqr lzma openexr png raqm tiff webp x xml zlib`.

I have a system where no package manager or similar is available. But I need HEIC support.

### Describe the solution you'd like

Is it possible to publish two appimages?
The current one as "lite/simple version" and a second one "full version" that support  almost all image formats/delegates?
Delegates for the "full version" which are missing in the "lite/simple version":
- Autotrace          --with-autotrace
- DPS                  --with-dps
- FFTW               --with-fftw
- FLIF                  --with-flif
- FlashPIX           --with-fpx
- Ghostscript lib   --with-gslib
- Graphviz           --with-gvc
- HEIC                 --with-heic
- LTDL                 --with-ltdl
- Magick++          --with-magick-plus-plus
- OpenJP2          --with-openjp2
- PANGO            --with-pango
- PERL                --with-perl
- RAW                 --with-raw
- RSVG               --with-rsvg
- WMF                 --with-wmf

### Describe alternatives you've considered

In my opinion it is the easiest way to create two appimages.

**Comment by dlemstra** [comment]
In case anyone wants to pick this up this can be done in the `app-image.yml` file.

**Comment by btimbermont** [comment]
I would love this to be added

**Comment by btimbermont** [comment]
@dlemstra I added HEIC support to the AppImage in this PR: https://github.com/ImageMagick/ImageMagick/pull/6098

The PR is waiting for approval/feedback though. I'd love for someone to have a look, it's a very small change.

**Comment by btimbermont** [comment]
With PR  #6098 support for HEIC (and AVIF) is now added

**Comment by richard-llmnn** [comment]
Thank you @btimbermont and @dlemstra for your help :100: .
Because not all formats are added yet to the current AppImage, I will keep the issue open.

**Comment by btimbermont** [comment]
I added jp2 support in https://github.com/ImageMagick/ImageMagick/pull/6630
